x/crosschain/types/cctx.go

```diff
@@ -131,9 +135,15 @@ func (m *CrossChainTx) AddRevertOutbound(gasLimit uint64) error {
 	// if not specified, the sender address is used
 	// note: this option is current only support for EVM type chains
 	if m.ProtocolContractVersion == ProtocolContractVersion_V2 {
-		revertAddress, valid := m.RevertOptions.GetEVMRevertAddress()
-		if valid {
-			revertReceiver = revertAddress.Hex()
+		if chains.IsBitcoinChain(m.InboundParams.SenderChainId, []chains.Chain{}) {
+			if m.RevertOptions.RevertAddress != "" {
```

Comment:
Maybe, to play safe, we could still use same pattern as above, as a protection to any accidental invalid address?
```
revertAddress, valid := m.RevertOptions.GetBTCRevertAddress(m.InboundParams.SenderChainId)
if valid {
	revertReceiver = revertAddress
}
```

Reply:
If it's invalid, the cctx should rather abort actually because if a user want to specify a revert address it's likely because the funds can't be recovered with the sender

Reply:
Like, whenever user specified a `revert address`, it is very likely the original `sender` is a WRONG address to get refund (for example, not recoverable).  So, when `zetacore` receives a invalid address, it is safer to abort and we could manually refund if needed as the funds is under our control.

Reply:
https://github.com/zeta-chain/node/issues/3446